src/llm.py

```diff
@@ -1,24 +1,34 @@
-from transformers import pipeline
+from langchain_cohere import ChatCohere
+from langchain.chains import SequentialChain, LLMChain
+from langchain.prompts.chat import ChatPromptTemplate
+from decouple import config
+from utils import get_prompt_templates
 
-
-def llm_summarise(article):
-    """
-    Summarise article to have word count betweem 150 to 170.
+def run_summariser_censor(body):
+    '''
+    Summarises and censors toxic words within post body using Cohere LLM.
 
     Parameters:
-    article: string to be summarised
+    Post body to be summarised and censored.
 
     Returns:
-    String containing summary
-    """
-    summariser = pipeline("summarization", model="facebook/bart-large-cnn")
+    The summarised and cleaned text.
+    '''
+    llm = ChatCohere(cohere_api_key=config('COHERE_API_KEY'))
+    summariser_template, censor_template = get_prompt_templates()
 
-    summary_arr = summariser(article, max_length=170, min_length=150, do_sample=False)
-    summary_text = summary_arr[0]["summary_text"]
-    return summary_text
+    # create prompt and chain objects
+    summariser_prompt = ChatPromptTemplate.from_template(template=summariser_template)
```

Comment:
I see that the summariser_template and censor_template are constants, should we also be following the same upper case naming convention here as well?

Reply:
okay!